### DISCUSSION TITLE
[Request] Chocolatey

### DISCUSSION THREAD

**Discussion by robsonsobral** [hyperspacedev/hyperspace#180] [Ideas]
Hi! I just found hyperspace. Any chance of make it available on Chocolatey? https://chocolatey.org/courses/creating-chocolatey-packages

**Comment by alicerunsonfedora** [comment]
We could definitely take a look into adding it to Chocolatey as long as it fits into our existing workflow and doesn't heavily require the use of Windows to get the package in place. A good sum of our members works on macOS or Linux and would love to automate some of this via CI as we mainly pull from GitHub Actions when releasing Windows builds.

**Comment by robsonsobral** [comment]
Thank you, @alicerunsonfedora .

There are some documentations on how to [create your package](https://chocolatey.org/courses/creating-chocolatey-packages/installing-upgrading-and-uninstalling) and how to [update it automatically](https://chocolatey.org/courses/creating-chocolatey-packages/automatic-packaging).

Thank you for considering.

**Comment by alicerunsonfedora** [comment]
I think we might also want to consider adding this to WinGet now that Microsoft has announced this as an option: https://github.com/microsoft/winget-pkgs#submitting-a-package

**Comment by alicerunsonfedora** [comment]
Working on adding v1.1.3 to WinGet. See https://github.com/microsoft/winget-pkgs/pull/2234

**Comment by alicerunsonfedora** [comment]
microsoft/winget-pkgs#2234 has been merged. 🎉
